django/db/migrations/executor.py

```diff
@@ -146,6 +150,9 @@ def apply_migration(self, state, migration, fake=False, fake_initial=False):
                 # Alright, do it normally
                 with self.connection.schema_editor() as schema_editor:
                     state = migration.apply(state, schema_editor)
+                    logger.info('Applied migration %s on database %s', migration, self.connection.alias)
+        if fake:
+            logger.info('Faked applying migration %s on database %s', migration, self.connection.alias)
```

Comment:
Could we have 1 logging statement and vary the action name based on `fake`? e.g. `"Fake applied" if fake else "Applied"`